src/store/legacy.ts

```diff
@@ -0,0 +1,56 @@
+import { PersistConfig, createMigrate, KEY_PREFIX, getStoredState } from 'redux-persist';
```

Comment:
Maybe a better name for this file would be `legacyStorage.ts` or `legacyPersistConfig.ts`

Reply:
Renamed it to `legacyAsyncStorage`